pipe/tests/test_reduce.py

```diff
@@ -1,5 +1,47 @@
+import os
+from tempfile import TemporaryDirectory
+from shutil import copy
 
-def test_import():
-    from ..reduce import resample_imagette_time
 
-    assert True
\ No newline at end of file
+def test_end_to_end():
+    """
+    Run locally with:
+    python -c "from pipe.tests.test_reduce import test_end_to_end as f; f()"
+    """
+    psflib = os.path.join(
+        os.path.dirname(__file__), os.path.pardir,
+        'data', 'eigenlib_815_281_70_0.pkl'
+    )
+    with TemporaryDirectory() as tempdir:
```

Comment:
This bit of magic creates a temporary directory in which all of the simulated data will be stored during the testing. `tempdir` becomes the path to the temporary directory.